File: iosApp/iosApp/Presentation/Main/CafeManagement/ExternalLink/ExternalLinkUiState.swift
//
//  ExternalLinkUiState.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/11.
//

import Foundation

struct ExternalLinkUiState {
    let title: String
    let url: String

    var displayTitle: String {
        title.isEmpty ? "외부 링크" : title
    }
}

File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/cafemanagement/externallink/ExternalLinkUiState.kt
package com.hhp227.concafe.presentation.main.cafemanagement.externallink

data class ExternalLinkUiState(
    val title: String,
    val url: String
) {
    val displayTitle: String
        get() = title.ifBlank { "외부 링크" }
}
